news/2019-07-07-beatmap-spotlights-spring-2019.md

```diff
@@ -0,0 +1,327 @@
+---
+layout: post
+title: "Beatmap Spotlights: Spring 2019"
+date: 2019-07-07 06:00:00 +0000
+---
+
+The sun is shining brightly and it's hot outside, so you shouldn't be outside during these dangerous times!  Instead we have prepared a nice package of awesome mapsets from last season for you to spend some time with. Come take a look and earn a fancy new medal for yourself by playing them all!
+
+![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/header.jpg)
+
+The beatmap charts for the Spring Spotlights can be found [here](https://osu.ppy.sh/rankings/taiko/charts?spotlight=263). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Spring 2019 Spotlights rankings will end on the 15th of September, 2019.**
+
+## Winter Spotlights results
+
+First of all, congratulations to the winners of the Seasonal Spotlights: Winter 2019. They will receive 3, 2, and 1 month of supporter for 1st, 2nd, and 3rd place respectively for their efforts!
+
+- **osu!:** [Toy](https://osu.ppy.sh/users/2757689), [Apraxia](https://osu.ppy.sh/users/4194445) and [xxluizxx47](https://osu.ppy.sh/users/4687701)
+- **osu!taiko:** [Jaye](https://osu.ppy.sh/users/4841352), [AlvaroYL](https://osu.ppy.sh/users/6333166) and [tasuke912](https://osu.ppy.sh/users/2774767)
+- **osu!catch:** [rostld](https://osu.ppy.sh/users/4365562), [RAMPAGE88](https://osu.ppy.sh/users/448547) and [-Nirfu-](https://osu.ppy.sh/users/7332050)
+- **osu!mania:** [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), [Stink God](https://osu.ppy.sh/users/7381289) and [Cryolien](https://osu.ppy.sh/users/1626983)
+
+
+# Navigation
+
+- <a href="#osu">osu!</a>
+- <a href="#taiko">osu!taiko</a>
+- <a href="#catch">osu!catch</a>
+- <a href="#mania">osu!mania</a>
+
+# Spring 2019 Spotlights
+
+## <a name="osu" id="osu"></a>osu!
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/kagefumi.jpg)](https://osu.ppy.sh/beatmapsets/910454)
+
+A mapper who is reputable in the community for their unorthodox music taste and style, [squirrelpascals](https://osu.ppy.sh/users/6151332), brings forth a very fitting interpretation of Hyakkei's "kagefumi". 
+
+This mapping contains unique stream patterns, minimalistic slider velocities, characterized slider shapes, and much more. squirrelpascals underlines the core portions of this chill, tranquil, math rock song with clarity and accentuates their features fantastically through distinctive design choices that align with the song's simplicity.
```

Comment:
From unique stream patterns, minimalistic slider velocities, characterized slider shapes, and much more, squirrelpascal underlines the core portions of this chill, tranquil, math rock song with clarity and accentuates their features fantastically through distinctive design choices that align with the song's simplicity.